include/extractor/scripting_environment_lua.hpp

```diff
@@ -68,6 +68,11 @@ class LuaScriptingEnvironment final : public ScriptingEnvironment
                     tbb::concurrent_vector<boost::optional<InputRestrictionContainer>>
                         &resulting_restrictions) override;
 
+    void setExtractionContainers(ExtractionContainers &extractor) override;
+
+    // FIXME This is static because I don't know how to access to this in lua nodeRef class binding.
+    static ExtractionContainers *extraction_containers;
```

Comment:
But why do we need the full extraction containers here.
The ScriptingEnvironment only needs to access the cache, right?